### PR TITLE
Added Comments for defaults

### PR DESCRIPTION
**Pull request by dmurko**
Adding comments so it's easier to read.

Moving to localizations:

```
gmt_offset: ''
blog_charset: 'UTF-8'

# Checkout endpoints
woo/woocommerce_checkout_pay_endpoint: 'order-pay'
woo/woocommerce_checkout_order_received_endpoint: 'order-received' 
woo/woocommerce_myaccount_add_payment_method_endpoint: 'add-payment-method'
woo/woocommerce_myaccount_delete_payment_method_endpoint: 'delete-payment-method'
woo/woocommerce_myaccount_set_default_payment_method_endpoint: 'set-default-payment-method'
woo/woocommerce_myaccount_orders_endpoint: 'orders'
woo/woocommerce_myaccount_view_order_endpoint: 'view-order'
woo/woocommerce_myaccount_downloads_endpoint: 'downloads'
woo/woocommerce_myaccount_edit_account_endpoint: 'edit-account'
woo/woocommerce_myaccount_edit_address_endpoint: 'edit-address'
woo/woocommerce_myaccount_payment_methods_endpoint: 'payment-methods'
woo/woocommerce_myaccount_lost_password_endpoint: 'lost-password'
woo/woocommerce_logout_endpoint: 'customer-logout'

# Permalinks

woo/woocommerce_permalinks:
    product_base: '/shop'
    category_base: 'product-category'
    tag_base: 'product-tag'
    attribute_base: ''
    use_verbose_page_rules: 'true'
```

Is based on Wizard Selection

```
woo/woocommerce_allowed_countries: 'all' # Selling location(s)
woo/woocommerce_product_type: 'physical' # 
```

Removed

```
# Page IDs TODO - aren't these generated on the fly?
woo/woocommerce_cart_page_id: ''
woo/woocommerce_checkout_page_id: ''
woo/woocommerce_myaccount_page_id: ''
woo/woocommerce_terms_page_id: ''
```

Unneeded

```
default_comments_page: 'newest' 
woo/woocommerce_email_footer_text: '{site_title}'
woo/woocommerce_checkout_highlight_required_fields: 'yes'
woo/woocommerce_catalog_columns: '3'
woo/woocommerce_catalog_rows: '4'
```

What?

```
woo/woocommerce_queue_flush_rewrite_rules: 'no'
woo/woocommerce_allow_tracking: 'no'
woo/woocommerce_api_enabled: 'no'
woo/woocommerce_admin_notices:
    - no_shipping_methods
```

### CODE REVIEW

File: defaults.yaml [hunk 1]
```diff
@@ -1,76 +1,78 @@
...
-woo/woocommerce_checkout_highlight_required_fields: 'yes'
-woo/woocommerce_checkout_order_received_endpoint: order-received
-woo/woocommerce_checkout_page_id: '8'
-woo/woocommerce_checkout_pay_endpoint: order-pay
-woo/woocommerce_default_customer_address: geolocation
-woo/woocommerce_delete_inactive_accounts:
+# WordPress
+fresh_site: '0' # Remove new WordPress notifications
+default_role: 'customer' # New User Default Role
+users_can_register: '0' # TODO Shouldn't this be 1?
```
Comment (dz0ny) [inline defaults.yaml]: They are added via WooCommerce, this indicates they can register on site.

File: defaults.yaml [hunk 2]
```diff
@@ -1,76 +1,78 @@
...
+# Account & Privacy
+woo/woocommerce_enable_guest_checkout: 'yes' # Allow customers to place orders without an account
+woo/woocommerce_enable_checkout_login_reminder: 'yes' # Allow customers to log into an existing account during checkout
+woo/woocommerce_enable_signup_and_login_from_checkout: 'yes' # Allow customers to create an account during checkout
+woo/woocommerce_enable_myaccount_registration: 'yes' # Allow customers to create an account on the "My account" page
+woo/woocommerce_registration_generate_username: 'yes' # When creating an account, automatically generate a username from the customer's email address
+woo/woocommerce_registration_generate_password: 'yes' # When creating an account, automatically generate an account password
+
+# Personal data retention (GDPR)
+woo/woocommerce_erasure_request_removes_order_data: 'yes' # Remove personal data from orders
```
Comment (dz0ny) [inline defaults.yaml]: woo/woocommerce_allow_tracking: 'no'

**Comment by dmurko** [comment]
## Review

Add under GDPR:
woo/woocommerce_allow_tracking: 'no'

Add somewhere under settings (slugs):
woo/woocommerce_queue_flush_rewrite_rules: 'yes' # Leave as-is, used for flushing rules

Add somewhere:
woo/woocommerce_api_enabled: 'no' # Leave as-is, used for REST API
woo/woocommerce_email_footer_text: '{site_title}' # Leave as-is, created by Store Wizard
woo/woocommerce_checkout_highlight_required_fields: 'yes' # Highly recommended

woo/woocommerce_allowed_countries: 'all' # Selling location(s), configured by Store Wizard
woo/woocommerce_product_type: 'physical' # Type of products sold, configured by Store Wizard

**Comment by dmurko** [comment]
@dz0ny Please review again.

**Pull request merged**